tests/test_mlp.py

```diff
@@ -31,6 +33,14 @@
             ]
             TEST_CASE_MLP.append(test_case)
 
+# test different activation layers
+TEST_CASE_ACT = []
+for act in ["GELU", "GEGLU", ("GELU", {"approximate": "tanh"}), ("GEGLU", {})]:
```

Comment:
without the changes, `("GEGLU", {})` cannot pass tests because of mismatch output features of linear1 cc @KumoLiu